rtf_vis_tool/src/Components/FrontendSummary.js

```diff
@@ -0,0 +1,63 @@
+/* Component to render Frontend Summary metrics. Specifically rotation sucesses, translation
+successes, pose successes, and correspondence inliers.
+
+Author: Adi Singh
+*/
+import React from "react";
+
+// Local Imports.
+import '../stylesheets/FrontendSummary.css'
+
+function FrontendSummary(props) {
+    /*
+    Args:
+        props.json.rotation.success_count (int): Number of successes from rotation averaging.
```

Comment:
these are actually not pertaining to rotation averaging (just the rotation from the 2-view relative rotation)

Reply:
"Number of image pairs with estimated relative rotation error underneath the threshold"
later, we should save the threshold also